hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeFileSplit.java

```diff
@@ -140,7 +132,7 @@ public Integer answer(InvocationOnMock invocationOnMock) throws Throwable {
 
       @Override
       public Void answer(InvocationOnMock invocation) throws Throwable {
-        byte[] bytes = invocation.getArgumentAt(0, byte[].class);
+        byte[] bytes = invocation.getArgument(0);
```

Comment:
the new API refers getArgument()
https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/invocation/InvocationOnMock.html#getArgument-int-